packages/elements/src/photon-multirx-form/photon-prescribe-workflow.tsx

```diff
@@ -223,7 +226,9 @@ export function PrescribeWorkflow(props: PrescribeProps) {
     const inProgressDraftedPrescriptionTreatmentId = props.formStore.treatment?.value?.id;
 
     // and then get the ones already added to the order (but not persisted)
-    const draftedPrescriptions = [...props.formStore.draftPrescriptions.value];
+    const draftedPrescriptions: ScreenablePrescription[] = draftPrescriptions().map(
```

Comment:
is there no way to do this business logic outside of the elements folder? my dream world elements is just used to arrange the components